Assignment/Program.c

```diff
@@ -1,14 +1,119 @@
-// ------------------------------------------------------------------------------------------------
+﻿// ------------------------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
 // Vinitha Perumal - GET
 // ------------------------------------------------------------------
 // Program.c
-// Program on main branch.
+// Program on A6 branch.
 // ------------------------------------------------------------------------------------------------
 #include <stdio.h>
-#include "Program.h"
+#include <io.h>
+#include <fcntl.h>
+
+/// <summary>To print the chessboard to the given output file </summary>
+void PrintChessBoard (FILE* output);
+
+/// <summary>To compare the output file with a reference file </summary>
+int CompareFiles (const char* outputFile, const char* referenceFile);
+
+/// <summary>To print a character(or an empty space) to both the output file and the console </summary>
+void PrintCell (FILE* output, wchar_t piece);
+
+void PrintCell (FILE* output, wchar_t character) {

```

Comment:
PrintCell is not appropriate name for the method, it can be PrintChar or PrintCharacter